src/robot/libraries/String.py

```diff
@@ -19,6 +19,8 @@
 from fnmatch import fnmatchcase
 from random import randint
 from string import ascii_lowercase, ascii_uppercase, digits
+from os.path import isfile, isabs, sep
```

Comment:
- `sep` is officially exposed directly via `os`, not via `os.path`. Should also just `import os` and then use 'os.sep`.
- Could also consider just importing `os.path` and using `os.path.isfile`, but also this style is fine as well.